CLDR-4152 show alternate locale in info panel

srl295 opened the pull request:
CLDR-4152

- [ ] This PR completes the ticket.

<!--
Thank you for your pull request.
Please see https://cldr.unicode.org/index/process for general
information on contributing to CLDR.

1. Make sure the ticket is filed at
https://unicode-org.atlassian.net/projects/CLDR/
2. Update the PR title and first line of this
message to include the ticket ID (CLDR-_____)
3. You will be automatically asked to sign the contributors’
license before the PR is accepted.
- sign: https://cla-assistant.io/unicode-org/cldr
- license: https://www.unicode.org/copyright.html#License
-->

ALLOW_MANY_COMMITS=true

srl295 commented:
https://github.com/unicode-org/cldr/assets/855219/3b43773d-bf2d-40e7-8f9d-95752b74ce0f

srl295 commented:
maybe we should have a switch, 'CLDR Labs' that lets you try out new experimental features.

DavidLRowe commented:
Just dreaming, but I think that I'd find it more useful to have a control (button or whatever) that swapped the values in the column from English to the selected reference language (French or Spanish or whatever). Then I could see (at the same time) how all the months were translated from English to the reference language, rather than one by one.

srl295 commented:
> Just dreaming, but I think that I'd find it more useful to have a control (button or whatever) that swapped the values in the column from English to the selected reference language (French or Spanish or whatever). Then I could see (at the same time) how all the months were translated from English to the reference language, rather than one by one.

That's a good point and a good reason to add an additional column, thanks.  I added it this way because it was easier to prototype- the general edit view is still not 'modernized'. 

I added a comment on the issue https://unicode-org.atlassian.net/browse/CLDR-4152?focusedCommentId=173802 which notes that only the English is fully vetted.  So, an additional column, or another row in the 'English' column might make sense, but I don't think it would be a good idea to actually hide the English.
